metricbeat/module/kubernetes/state_service/_meta/data.json

```diff
@@ -1,50 +1,47 @@
 {
-    "@timestamp": "2019-11-27T18:13:26.661Z",
+    "@timestamp": "2019-12-18T14:47:52.427Z",
```

Comment:
Interesting this actually uses the current timestamp. In x-pack/aws, seems like there is a default timestamp: https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/aws/billing/_meta/data.json#L2

Reply:
I launch metricbeat with debug output and copy data.json from there.
Have been doing that for a while, probably there is some other way of obtaining that file.

Reply:
Hmm for most of the modules/metricsets, I see a TestData function that generates `data.json` like https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/aws/billing/billing_integration_test.go#L16

Reply:
oh, thanks for sharing!